### ISSUE TITLE
Apply Script editor and certain feilds are empty and uneditable

### ISSUE THREAD

**Issue by CyborgArmGun** [TES5Edit/TES5Edit#1019]
Version: xEdit 4.1.4b EXTREMELY EXPERIMENTAL (problem persists with xEdit 4.0.4b )

Blank fields in the text Editor Portion of Apply Script, and later on within a texture path input box.

1. Go to 'Apply script"
2. Lower box is empty and blank, and you cannot select or click anything.
3. Selecting the "Nif - batch textures replacer"
4. lower box text field is blank and unable to be interacted with.

Expected behavior:
Text box has the Script code, Usually gives a popup if I accidentally click on it if I want to save. 

Screenshots:

https://i.imgur.com/yCfee1E.png | https://i.imgur.com/Z8S4DFg.png -|- https://i.imgur.com/pIl7CrH.png | https://i.imgur.com/dADBYmR.png

Additional context:
Without the ability to provide input, some scripts are rendered useless.

Edition	Windows 11 Home
Version	21H2
Installed on	‎12/‎21/‎2021
OS build	22000.708
Experience	Windows Feature Experience Pack 1000.22000.708.0
System type	64-bit operating system, x64-based processor

**Comment by VlitS** [comment]
If the fields in the windows brought up by a script are also uneditable, then you have some  weird/broken configuration on your computer. You could try renaming/deleting the settings file (%localappdata%\Fallout4, plugins.fo4viewsettings *), and see if t hat fixes the issue, but I am guessing you have weird permissions issues. If you need help narrowing things down (should wiping your settings file not fix it), I recommend checking out the discord (link in the top right of xEdit). 

Tested with 4.0.4b, works fine for me. 

*With appropriate folder and file name change for other games

**Comment by CyborgArmGun** [comment]
I deleted the Skyrim settings so I'll try the Fo4 ones as well.

Deleting The FO4 settings did jack squat.

I posted on discord before this and they completely ignored me so my expectations are pretty nonexistent.

**Comment by jbostrus** [comment]
You really didn't give much information, or your questions, on Discord.

I was able to reproduce this issue but only with 4.1.4b (edit: both 32 and 64bit). This looks to be related to TSynMemo changes and the issues of #994 which I've submitted patches for. It doesn't seem to resolve it for TMemo that scripts create though. So only the initial script edit window works as previously designed with the patches.
